include/robot_interfaces/status.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <robot_interfaces/loggable.hpp>
+
+namespace robot_interfaces
+{
+
+  struct Status : public Loggable
```

Comment:
Indentation is not following the style guide (4 spaces).

Reply:
indeed